### PR TITLE
cookie

### PR DESCRIPTION
**Pull request by PyGamer0**
cookie

### CODE REVIEW

**Review by Lyxal — APPROVED** [review]

**Review by AMiller42 — APPROVED** [review]
cookie

**Comment by Lyxal** [comment]
cookie

**Review by Lyxal — APPROVED** [review]
cookie

File: hooks/msghooks.py [hunk 1]
```diff
@@ -115,6 +115,11 @@ def receive_message():
                           /,/
                           |/`)
                           'c=,"""
+        if re.match(r"(make|pls make|plz make|please make) (me | meh) a (cookie|cokie)", without_ping):
```
Comment (ysthakur) [inline hooks/msghooks.py:118]: This regex looks not epic. Maybe try `(pl(s|z|ease) )?make meh? a coo?kie`?

File: hooks/msghooks.py [hunk 2]
```diff
@@ -115,6 +115,11 @@ def receive_message():
...
+            if random.random() <= 0.5 or message["user_id"] == 513493:
```
Comment (ysthakur) [inline hooks/msghooks.py:119]: Mmm, I'd prefer more choices here.

File: hooks/msghooks.py (hunk 1 above)
Comment (ysthakur) [reply hooks/msghooks.py:118]: It'd be nice if it also gave you things other than cookies (biscuits, ice cream, money, coke, etc.)

**Review by ysthakur — CHANGES_REQUESTED** [review]
cookie

**Comment by ysthakur** [comment]
#40

File: hooks/msghooks.py (hunk 1 above)
Comment (PyGamer0) [reply hooks/msghooks.py]: cookie

File: hooks/msghooks.py (hunk 2 above)
Comment (PyGamer0) [reply hooks/msghooks.py:119]: ?

Comment (ysthakur) [reply hooks/msghooks.py:119]: @PyGamer0  Like ice cream, money, baguettes, etc.

File: hooks/msghooks.py [hunk 3]
```diff
@@ -115,7 +115,7 @@ def receive_message():
                           /,/
                           |/`)
                           'c=,"""
-        if re.match(r"(make|pls make|plz make|please make) (me | meh) a (cookie|cokie)", without_ping):
+        if re.match(r"(pl(s|z|ease) )?make meh? a coo?kie", without_ping):
```
Comment (ysthakur) [inline hooks/msghooks.py:118]: I am happy with the cookie

**Review by hyper-neutrino — APPROVED** [review]
cookie

**Comment by nailuj29gaming** [comment]
> too heated

**Comment by Lyxal** [comment]
yes.

**Comment by ysthakur** [comment]
cookie

**Pull request merged**